website/Makefile

```diff
@@ -7,6 +7,8 @@ build:
 		--rm \
 		--tty \
 		--volume "$(shell pwd):/website" \
+		--volume "/website/source/docs/guides" \
+		--volume "$(shell pwd)/source/docs/guides/index.html.md:/website/source/docs/guides/index.html.md" \
```

Comment:
Should we maybe describe why we're doing this in a comment somewhere?

Reply:
Ah yeah, I had it committed locally but forgot to push the note 🤦‍♂